src/Krank/Checkers/IssueTracker.hs

```diff
@@ -65,7 +65,7 @@ serverDomain (Gitlab (GitlabHost h)) = h
 gitRepoRe :: RE.Regex
 -- NOTE: \b at the beginning is really import for performances
 -- because it dramatically reduces the number of backtracks
-gitRepoRe = [RE.re|\b(?>https?://)?(?>www\.)?([^/ ]+)/([^/]+)/((?>[^/]+))/issues/([0-9]+)|]
+gitRepoRe = [RE.re|\b(?>https?://)?(?>www\.)?([^/ ]+)/(.*)/([^-][^/]*)(?>/-)?/issues/([0-9]+)|]
```

Comment:
Could you try to replace the `.*` by something similar to `[^/]+`. I also need to test performances of this.

Reply:
well that the whole point. it's not `[^/]`. I need to capture slashes because of the hierarchical gitlab group URL structure
I need to capture everything till the last part of the URL. I don't have a super idea to do that.

Reply:
Good point!